### PR TITLE
postgresql: add support for debugger (used by pgadmin)

### PR DESCRIPTION
**Pull request by backerman**
pgadmin (in casks) SQL debugging support requires a module that's maintained separately from core PostgreSQL; this change adds an option that installs the module.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/postgresql.rb [hunk 1]
```diff
@@ -22,6 +23,13 @@ class Postgresql < Formula
   depends_on "openssl"
   depends_on "readline"
   depends_on "libxml2" if MacOS.version <= :leopard # Leopard libxml is too old
+
+  # The debugger module is shipped separately.
+  resource "debugger" do
+    url "git://git.postgresql.org/git/pldebugger.git",
+      :revision => "85d7b3b2821301e182d5974d9e6f353d7a241eff"
```
Comment (DomT4) [inline Library/Formula/postgresql.rb]: Is there a stable tag to this or?

File: Library/Formula/postgresql.rb [hunk 2]
```diff
@@ -80,6 +88,18 @@ def install
...
                                     "libdir=#{lib}",
                                     "pkglibdir=#{lib}/postgresql"
+    if build.with? "debugger"
+      resource("debugger").stage("#{buildpath}/contrib/pldebugger")
+      cd "contrib/pldebugger" do
+        system "make"
+        system "make", "install", "datadir=#{pkgshare}",
+                                  "libdir=#{lib}",
+                                  "pkglibdir=#{lib}/postgresql"
+        File.open("#{pkgshare}/postgresql.conf.sample", "a") do |f|
```
Comment (DomT4) [inline Library/Formula/postgresql.rb]: If this file already exists use:
```ruby
(pkgshare/"postgresql.conf.sample").append_lines <<-EOS.undent
```
and if it doesn't use:
```ruby
(pkgshare/"postgresql.conf.sample").write  <<-EOS.undent

**Comment by apjanke** [comment]
The test for `madlib` [failed on 10.9 and 10.10](http://bot.brew.sh/job/Legacy%20Homebrew%20Pull%20Requests/44099/)

```
==> /usr/local/Cellar/madlib/1.8/bin/madpack -p postgres -c brew/@localhost/test_madpack install
madpack.py : INFO : Detected PostgreSQL version 9.5.
madpack.py : ERROR : This version is not among the PostgreSQL versions for which MADlib support files have been installed (9.4).
```

The `madlib` caveats say

```
  def caveats; <<-EOS.undent
    MADlib must be rebuilt if you upgrade PostgreSQL:

      brew reinstall madlib
    EOS
  end
```
so maybe this is a revision bump that we missed last time postgresql was upgraded.

Madlib:

```
commit 705c74c2f156116f4b6146f6ea954bf84d50cdf8
Author: BrewTestBot <brew-test-bot@googlegroups.com>
Date:   Sat Aug 1 07:12:14 2015 +0100

    madlib: update 1.8 bottle.
```

Postgresql:
```
commit c41ee814a3da12fc508b79fd691f6f97c26cb165
Author: BrewTestBot <brew-test-bot@googlegroups.com>
Date:   Fri Apr 1 11:15:41 2016 +0100

    postgresql: update 9.5.2 bottle.
```

I'll put in a bottle revision bump for `madlib`.

**Comment by apjanke** [comment]
Closing this due to inactivity/lack of response. If you'd still like to pursue this, feel free to respond to the previous comments/questions and re-open on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core).

**Pull request closed without merging**